extensions/wikia/Recirculation/RecirculationController.class.php

```diff
@@ -52,14 +52,30 @@ public function discussions() {
 	}
 
 	public function footer() {
-		global $wgSitename;
+		global $wgSitename, $wgCityId;
 
 		$themeSettings = new ThemeSettings();
+		$discussionsEnabled =
+		$wgEnableDiscussions = WikiFactory::getVarValueByName( 'wgEnableDiscussions', $wgCityId );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$wgEnableDiscussions" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)